package.json

```diff
@@ -27,9 +27,9 @@
     "win32"
   ],
   "dependencies": {
+    "@mapbox/node-pre-gyp": "^1.0.5",
```

Comment:
may I suggest you to use alias here:

https://stackoverflow.com/questions/56134857/how-to-install-npm-package-under-alias-or-different-name

Reply:
I'd rather not – `node-pre-gyp` is simply not maintained anymore, I don't see a good reason to confuse people with an alias syntax :)